cyclops-ui/src/components/k8s-resources/DaemonSet.tsx

```diff
@@ -1,15 +1,22 @@
 import React, { useEffect, useState } from "react";
-import { Col, Divider, Row, Alert } from "antd";
+import { Col, Divider, Row, Alert, Button } from "antd";
 import axios from "axios";
 import { mapResponseError } from "../../utils/api/errors";
 import PodTable from "./common/PodTable";
 
 interface Props {
-  name: string;
-  namespace: string;
+  resource: {
+    group: string;
+    kind: string;
+    version: string;
+    name: string;
+    namespace: string;
+  };
```

Comment:
Is there a reason to add group, version and kind here? Can we leave only name and namespace in the props

Reply:
Hey @petar-cvit what I felt was that group api and version can change and also instead of sending each of this 5 prop (group,api,version,name,namespace) we can send the resource and we can use it

Reply:
group, version and kind are not likely to change for these resources. There is also group, version and kind validation on the backend side which is expecting the values hardcoded here. Thanks for the concern!

Reply:
Got you, so only sending name and namespace prop is sufficient for Deamonset right?

Reply:
Yup. Name and namespace is enough for all of those (Deployment, StatefulSet, DaemonSet)